features/order/managing_orders/modifying_shipping_address/modifying_shipping_address.feature

```diff
@@ -0,0 +1,75 @@
+@modifying_shipping_address
+Feature: Modifying a customer shipping address after an order has been placed
+    In order to ship an order to a correct place
+    As an Administrator
+    I want to be able to modify a customer's shipping address after an order has been placed
+
+    Background:
+        Given the store operates on a single channel in the "United States" named "Web"
+        And that channel uses the "USD" currency by default
```

Comment:
Yes, we have, but I wanted to clearly see what is our currency, because we modify it later.